setup.py

```diff
@@ -103,10 +103,11 @@ def txt_read(*paths):
 setup_args['install_requires'] =\
     [
         "numpy",
-        "matplotlib",
+        "matplotlib>=1.5",
+        "cartopy",
```

Comment:
@QuLogic Cool! But where is this documented? I have just seen this file: https://github.com/SciTools/cartopy/blob/v0.13.x/requirements/plotting.txt 

Is this mechanism ``[basename_under_requirements_folder]`` a standard feature of ``distutils``?